docs/tutorial.js.html

```diff
@@ -267,10 +272,10 @@ <h1 class="page-title">Source: tutorial.js</h1>
     let nextStep;
     if (step) {
       if (typeof step === 'number') {
-        if (step &lt; 0 || step >= this.steps.length) {
+        if (step &lt; 1 || step > this.steps.length) {
```

Comment:
I think using one-indexed step number is more consistent with the `stepNum(step)` function.
Nothing inside Chariot is passing a step number into this function at the moment, but this could be a breaking change for library users.